content/MBC3.md

```diff
@@ -16,6 +18,12 @@ supported now.
 
 ### A000-BFFF - RAM Bank 00-03, if any (Read/Write)
 
+Depending on the current Bank Number/RTC Register selection (see below),
+this memory space is used to access an 8KByte external RAM Bank, or a
```

Comment:
I think the style agreed on is "8 KB"

Reply:
No
Please, see https://github.com/gbdev/pandocs/wiki/Document-Style

Reply:
So KiB, then. Sorry!

Reply:
Done